src/Sylius/Behat/Context/Api/Shop/LoginContext.php

```diff
@@ -55,6 +75,30 @@ public function iAmAVisitor(): void
         // Intentionally left blank;
     }
 
+    /**
+     * @When I am a logged in customer with email :email
+     */
+    public function iAmALoggedInCustomerWithEmail(string $email): void
+    {
+        $this->shopAuthenticationTokenClient->request(
+            'POST',
+            '/api/v2/shop/authentication-token',
+            [],
+            [],
+            ['CONTENT_TYPE' => 'application/json', 'HTTP_ACCEPT' => 'application/json'],
+            json_encode(['email' => $email, 'password' => 'sylius'])
+        );
+
+        $response = $this->shopAuthenticationTokenClient->getResponse();
+        $content = json_decode($response->getContent(), true, 512, \JSON_THROW_ON_ERROR);
+
+        Assert::keyExists(
+            $content,
+            'token',
+            SprintfResponseEscaper::provideMessageWithEscapedResponseContent('Token not found.', $response)
```

Comment:
Thinking out loud, is it needed here? I don't mean is message needed but this construction with SprintResponseEscaper? We can just return `Token not found.` imo